### PR TITLE
Adding version of code that includes H-R diagram stuff

### PR DESCRIPTION
**Pull request by drewjrosen**
Adding code that calculates the color-magnitude H-R diagram using physical distances from Gaia matched with 2Mass matched objects.

### CODE REVIEW

File: highsnobjects.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+
+
+
+#calcilate color G-J
+j_k=j_band[highSNindices]-k_band[highSNindices]
+
+#calculate absolute magnitude
+distance_pc=distance*10**3.
+
```
Comment (MishaSavchenko) [inline highsnobjects.py]: unnecessary empty lines

**Review by MishaSavchenko — APPROVED** [review]
Some unnecessary empty lines here and there other than that the code looks solid 👍

File: highsnobjects.py (hunk 1 above)
Comment (zhiyanwang) [reply highsnobjects.py]: Excellent.

**Review by zhiyanwang — APPROVED** [review]
Excellent

**Pull request merged**